src/index.css

```diff
@@ -2,29 +2,276 @@
+  overflow: hidden;
+  top: 0;
+  left: 0;
+  right: 0;
+  border-bottom: 1px solid var(--OffWhite);
+  width: 100%;
+  background-color: var(--White);
+  z-index: 20;
+}
+
+.links-bar {
+  gap: 3rem;
+  align-items: center;
+}
+
+.links-bar h2 {
+  font-family: var(--Montserrat);
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
 }
 
-.navigation-bar ul {
+.links-bar ul {
   list-style: none;
   gap: 2rem;
 }
 
+.links-bar ul li {
+  opacity: 0.5;
+  font-family: var(--Montserrat);
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.links-bar ul li:hover {
+  letter-spacing: 2.3px;
+}
+
+.links:hover,
+.links:focus,
+.links:focus-within {
+  color: #e53636;
+}
+
+.navigation-bar svg {
+  color: #0290ff;
+  padding: 0.2rem;
+  border: solid 1px var(--OffWhite);
+  border-radius: 50%;
+  font-size: 1.5rem;
+}
+
 .books-container {
-  margin-top: 2rem;
-  gap: 2rem;
-  align-items: center;
+  justify-content: space-between;
+  margin: 1rem 4rem 0 4rem;
+  border: 1px solid var(--OffWhite);
+  padding: 2.5rem;
+}
+
+.book-item {
+  flex-direction: column;
+  width: 50%;
+}
+
+.category {
+  opacity: 0.5;
+  font-family: var(--Montserrat);
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.title {
+  font-family: var(--RobotoSlab);
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.author {
+  font-family: var(--RobotoSlab);
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin-top: 0.25rem;
+}
+
+.remove-btn {
+  border: none;
+  background-color: transparent;
+  padding: 0;
+  margin-top: 1.3rem;
+  width: 0;
+  font-family: var(--RobotoSlab);
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.progress-bar-div {
+  width: 3.25rem;
+  height: 3.25rem;
+  align-self: center;
+}
+
+.percentage-completed {
+  align-self: center;
+  display: flex;
+  gap: 0.7rem;
+  width: 50%;
+}
+
+.percent {
+  flex-direction: column;
+  padding-right: 3.5rem;
+  border-right: 1px solid var(--OffWhite);
+  align-self: center;
+}
+
+.number {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  color: #121212;
+}
+
+.completed {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  color: #121212;
+}
+
+.update-progress-div {
+  flex-direction: column;
+  padding-left: 3.5rem;
+}
+
+.current-chapter {
+  opacity: 0.5;
+  font-family: var(--RobotoSlab);
+  font-size: 0.813rem;
+  font-weight: 300;
+  color: #121212;
+}
+
+.particular-chapter {
+  font-family: var(--RobotoSlab);
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #121212;
+  margin-top: 0.4rem;
 }
 
 .text {
-  margin-top: 3rem;
+  font-family: var(--RobotoSlab);
+  font-size: 0.813rem;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--White);
```

Comment:
Your final design should match Zeplin's design.

- [ ] color of your font is not pure white and not the same as in the Zeplin project. Please check it and apply the same color;
- [ ] capitalize all fonts of your buttons with blue background (as the same as in the Zeplin);

Reply:
Hi @SarvarKh ,

I appreciate your feedback.

The fonts are colored according to the colors used in Zeplin. the `var(--White)` is for the hex code #fff which is white.

Thank you.